rest_framework/validators.py

```diff
@@ -138,7 +138,9 @@ def __call__(self, attrs):
         queryset = self.queryset
         queryset = self.filter_queryset(attrs, queryset)
         queryset = self.exclude_current_instance(attrs, queryset)
-        if queryset.exists():
+
+        # Ignore validation if any field is None
+        if None not in attrs.values() and queryset.exists():
```

Comment:
Wouldn't this theoretically fail is a null value was passed in for a field that was not part of the unique constraint?